pkg/controller/installation/core_controller.go

```diff
@@ -770,10 +786,25 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 			r.status.SetDegraded("Failed to get Elasticsearch license type", err.Error())
 			return reconcile.Result{}, err
 		}
+		elasticsearchSecret = &corev1.Secret{}
+		err = r.client.Get(ctx, types.NamespacedName{Name: render.ElasticsearchPublicCertSecret, Namespace: render.OperatorNamespace()}, elasticsearchSecret)
+		if err != nil && !apierrors.IsNotFound(err) {
+			log.Error(err, err.Error())
+			r.status.SetDegraded("Failed to get Elasticsearch pub cert secret", err.Error())
```

Comment:
This is a user-facing message, we shouldn't abbreviate here.

"Failed to get Elasticsearch public certificate"

Same for the Kibana one

Reply:
I'll address this and the other comments in a follow-up, thanks @caseydavenport !

Reply:
@lmm sounds good - please send me that PR when it's ready.